documentation/assemblies/upgrading/assembly-upgrade-resources.adoc

```diff
@@ -3,38 +3,40 @@
 // // assembly-upgrade.adoc
 
 [id='assembly-upgrade-resources-{context}']
-= Strimzi resource upgrades
+= Strimzi custom resource upgrades
```

Comment:
Maybe we can add here somewhere that this is part of the preparation to Kubernetes CRD version `v1` which needs to happen before Kubernetes 1.22.

Reply:
Added a note: 

> NOTE: Upgrade of the custom resources to `{ApiVersion}` prepares Strimzi for a move to Kubernetes CRD `v1`, 
> which is required for Kubernetes 1.22.